src/PrestaShopBundle/Resources/config/services/core/util/helper_card.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  _defaults:
+    public: true
+
+  prestashop.core.util.helper_card.team_helper_card_documentation_link_provider:
+    class: 'PrestaShop\PrestaShop\Core\Util\HelperCard\HelperCardDocumentationLinkProvider'
+    arguments:
+      - '@=service("prestashop.adapter.legacy.context").getContext().language.iso_code'
+      -
+        'en': 'http://doc.prestashop.com/display/PS17/Team'
+        'fr': 'http://doc.prestashop.com/pages/viewpage.action?pageId=51840334'
+        'es': 'http://doc.prestashop.com/display/PS17/Equipo'
+        'it': 'http://doc.prestashop.com/pages/viewpage.action?pageId=54266520'
+      - 'http://doc.prestashop.com/display/PS17/Team'
```

Comment:
im thinking about configuration for all card, like so:

```yaml
 arguments:
      - '@=service("prestashop.adapter.legacy.context").getContext().language.iso_code'
      -
       team:
          -        
            'en': 'http://doc.prestashop.com/display/PS17/Team'
            'fr': 'http://doc.prestashop.com/pages/viewpage.action?pageId=51840334'
            'es': 'http://doc.prestashop.com/display/PS17/Equipo'
            'it': 'http://doc.prestashop.com/pages/viewpage.action?pageId=54266520'
          - 'http://doc.prestashop.com/display/PS17/Team'
        meta:
          -        
            'en': 'http://doc.prestashop.com/display/PS17/SEO+and+URLs'
            'fr': 'http://doc.prestashop.com/display/PS17/SEO+et+URL'
            'es': 'http://doc.prestashop.com/display/PS17/SEO+y+URLs'
            'it': 'http://doc.prestashop.com/display/PS17/SEO+e+URL'
          - 'http://doc.prestashop.com/display/PS17/SEO+and+URLs'
```

and it the code `$helperCardDocumentationLinkProvider->getLink('team');` @matks wdyt?

Reply:
go for it :+1:

Reply:
+1